src/lib/scheduler/node_queue_scheduler.cpp

```diff
@@ -30,9 +32,11 @@ NodeQueueScheduler::~NodeQueueScheduler() {
 void NodeQueueScheduler::begin() {
   DebugAssert(!_active, "Scheduler is already active");
 
-  _workers.reserve(Hyrise::get().topology.num_cpus());
-  _queue_count = Hyrise::get().topology.nodes().size();
-  _queues.reserve(_queue_count);
+  _worker_count = Hyrise::get().topology.num_cpus();
```

Comment:
At first glance, I only find one use of `_worker_count` (which should also be redundant for `_workers.size()`). So simply use `_workers.size()` there?